test/packages/aws/_dev/build/docs/billing.md

```diff
@@ -0,0 +1,7 @@
+# billing
```

Comment:
Is each of these files like a README for the data stream? If so, should it be placed under `<package root>/data_stream/<data stream>/_dev/build/README.md` instead of `<package root>/_dev/build/docs/<data stream>.md`?

Reply:
Note that we could still have `elastic-package build` generate the resulting docs in `<package root>/docs/<data stream>.md`.

Reply:
Thank you @ycombinator !! I think these files are readmes for policy templates instead of data streams: https://github.com/elastic/package-spec/blob/master/versions/1/_dev/build/docs/spec.yml#L9

For example: I don't want to have separate READMEs for ec2 logs and ec2 metrics. That's why I put them under `<package root>/_dev/build/docs/<policy template>.md`. Is there a place for policy template doc specific?

Reply:
@ycombinator I think the argument raised by @kaiyan-sheng is valid, we may need to keep these .md files in a single place. I'm fine with having them together with package's README.md.

Reply:
Ah, thanks for clarifying that these are READMEs for policy templates, not data streams. In that case it doesn't make sense to place them under data stream folders.

However, I would still consider creating a folder hierarchy for policy templates, something like:
```
<package root>/
  policy_template/
    <policy template>/
```

This would be more future-proof, in case we decide to add more files related to policy templates. 

Then policy template READMEs could go under each `<policy template>/_dev/build/docs/README.md` (mirrors the structure of the package-level docs folder). The generated READMEs would go under `<package root>/docs/<policy template>.md` so all generated documentation is in one place.

Thoughts?

Reply:
I think I would be for simplicity here as I like the fact that all docs are in a single folder. Any preferences here, @ruflin?

Reply:
Would also keep it in a single directory for now to keep it simple but no strong preference.

Reply:
Okay, sounds like the consensus is to keep all README files in a single directory for now. Let's go with it!

Reply:
Thanks for all the input!! Will do!